src/Grpc.AspNetCore.Server/build/netcoreapp3.0/Grpc.AspNetCore.Server.targets

```diff
@@ -1,9 +0,0 @@
-<Project>
```

Comment:
We could keep this until 16.2 is GA so users using 16.1 will have DesignTimeBuilds by default.